README.md

```diff
@@ -1,49 +1,48 @@
+Since this tutorial will be using Google Cloud Platform (GCP), it's designed for
+those with some experience GCP. While no specialized GCP knowledge is required,
+the guide assumes knowledge of basic GCP concepts and terminology.
 
 ## Using
 
-You can follow this guide from within Google's Cloud Shell starting with [this link](https://console.cloud.google.com/cloudshell/open?cloudshell_image=gcr.io/graphite-cloud-shell-images/terraform:latest&cloudshell_git_repo=https://github.com/robin-norwood/terraform-getting-started-gcp-cloud-shell&cloudshell_git_branch=master&cloudshell_working_dir=tutorial/&open_in_editor=./main.tf&cloudshell_tutorial=./cloudshell_tutorial.md)
+To follow this tutorial, you will need a Google Cloud Platform (GCP) account. If
+you do not have a GCP account, [create one
+now](https://console.cloud.google.com/freetrial/). This tutorial can be
+completed using only the services included in the GCP [free
+tier](https://cloud.google.com/free/).
 
-To follow the guide, you'll need an active Google Cloud Platform account.
+You can follow this guide from within Google's Cloud Shell starting with [this link](https://console.cloud.google.com/cloudshell/open?cloudshell_image=gcr.io/graphite-cloud-shell-images/terraform:latest&cloudshell_git_repo=https://github.com/hashicorp/terraform-getting-started-gcp-cloud-shell&cloudshell_git_branch=master&cloudshell_working_dir=tutorial/&open_in_editor=./main.tf&cloudshell_tutorial=./cloudshell_tutorial.md)
 
-## Code
+You will be prompted to trust this image. Answer "Yes".
 
-The source code for this guide is hosted in [this GitHub repository](https://github.com "FIXME: Link to GH repo").
-
-## Building
+## Use a Custome Image
 
 This tutorial works fine with the image used above. However, the version of Terraform included in that image may not be the latest version. You can build a Docker image with the latest version instead if you prefer.
 
-### Build the docker image
+### Build the Docker Image
 
 1. Set up docker and the gcloud command line utility as described in the "Before you begin" section of the [GCP Container Registry Quickstart](https://cloud.google.com/container-registry/docs/quickstart "Container Registry Quickstart Documentation").
 1. Run: `docker build . -t terraform-gcp-gsg:v$(date "+%Y-%m-%d")`
 1. Optionally, inspect/test image - for example:
   `docker run -it --entrypoint /bin/sh terraform-gcp-gsg:v$(date "+%Y-%m-%d")`
 
-### Deploy docker image to the image registry
+### Deploy Docker Image the GCP Image Registry
 
 1. Make sure docker is configured to authenticate with gcloud:
   - `gcloud auth configure-docker`
 1. `docker tag terraform-gcp-gsg:v$(date "+%Y-%m-%d") gcr.io/[PROJECT-ID]/terraform-gcp-gsg:v$(date "+%Y-%m-%d")`
 
-### Using the docker image with Cloud Shell
+### Use the Docker Image with Cloud Shell
 
 1. Update the URL above to use the URL to your new docker image.
 1. You'll be prompted to trust this image. Answer "Yes".
-
-## Known Issues
```

Comment:
is there a reason you removed this?

Reply:
It's not really useful anymore - it talked about the google_project_services issue that's been fixed, and also a vague and not very helpful warning about cloud shell not working well if you have networking problems.